src/mon/PGMap.cc

```diff
@@ -870,8 +870,8 @@ void PGMapDigest::dump_cluster_stats(stringstream *ss,
     TextTable tbl;
     tbl.define_column("SIZE", TextTable::LEFT, TextTable::RIGHT);
     tbl.define_column("AVAIL", TextTable::LEFT, TextTable::RIGHT);
-    tbl.define_column("RAW USED", TextTable::LEFT, TextTable::RIGHT);
-    tbl.define_column("%RAW USED", TextTable::LEFT, TextTable::RIGHT);
+    tbl.define_column("USED", TextTable::LEFT, TextTable::RIGHT);
```

Comment:
might want to updated https://github.com/ceph/ceph/blob/master/doc/rados/operations/monitoring.rst accordingly. search for `RAW USED` in it.